socialhome/streams/app/components/ReactionsBar.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <div class="grid-item-bar d-flex justify-content-start">
+        <slot />
+        <div class="ml-auto grid-item-reactions mt-1">
+            <b-button
+                v-if="showShares"
+                :class="{'item-reaction-shared': hasShared$, 'item-reaction': true}"
```

Comment:
I believe you can do `class="item-reaction" :class="{'item-reaction-shared': hasShared$}"` .. ie both `class` and `:class` can exist without having to make it an "if" statement by giving an extra `true` value.

Reply:
I don't think so but it seems `:class="[{'item-reaction-shared': hasShared$},'item-reaction']"` will work.

Reply:
Sorry forgot to link docs, it does work and IMHO is clearer to define classes as `class` if they don't need to be bind.

> You can have multiple classes toggled by having more fields in the object. In addition, the v-bind:class directive can also co-exist with the plain class attribute. 

https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax

Reply:
> In addition, the `v-bind:class` directive can also co-exist with the plain `class` attribute.

Indeed.